co2eq/purchase/index.js

```diff
@@ -10,11 +10,27 @@ import {
   UNIT_ITEM,
   UNIT_LITER,
 } from '../../definitions';
-import { convertToEuro, getAvailableCurrencies } from '../../integrations/utils/currency/currency';
 import { getChecksum } from '../utils';
 import footprints from './footprints.yml';
 import consumerPriceIndex from './consumerpriceindices.yml';
 
+const exchangeRates2011 = require('./exchange_rates_2011.json');
+
+export function convertTo2011Euro(amount, currency) {
+  if (currency.toUpperCase() === 'EUR') return amount;
+
+  const exchangeRate2011 = exchangeRates2011.rates[currency.toUpperCase()];
+  if (!exchangeRate2011) throw new Error(`Unknown currency '${currency}'`);
+  return amount / exchangeRate2011;
+}
+
+export function getAvailableCurrencies2011() {
+  return [...Object.keys(exchangeRates2011.rates), 'EUR'];
```

Comment:
Not sure about the usefulness of this function, 'EUR' is already in the keys of the object, so it is the same as `Object.keys(exchangeRates2011.rates)` But nice use of array destructuring :)